libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/NavigationViewListener.kt

```diff
@@ -131,6 +131,15 @@ abstract class NavigationViewListener {
      */
     open fun onInfoPanelSettling() = Unit
 
+    /**
+     * Called when the info panel is being dragged.
+     *
+     * @param slideOffset The new offset of info panel within [-1,1] range. Offset increases
+     *     as info panel is moving upward. From 0 to 1 the panel is between collapsed and
+     *     expanded states and from -1 to 0 it is between hidden and collapsed states.
+     */
```

Comment:
```suggestion
     * @param slideOffset The new offset of info panel within [-1,1] range. Offset increases
     *  as info panel is moving upward and decreases in the opposite direction. 
     *  Range 0 to 1: Denotes the panel is transitioning from collapsed to expanded state. 
     *  Range -1 to 0: Denotes the panel is transitioning from hidden to collapsed state.
     */
```

Reply:
Are you sure you want to rephrase this? I borrowed this description from https://github.com/material-components/material-components-android/blob/1ef53e2b37e6f5c14e49ada180dc2c0306e58e95/lib/java/com/google/android/material/bottomsheet/BottomSheetBehavior.java#L109

Reply:
Yes please.